.github/workflows/mirror.yml

```diff
@@ -37,11 +38,93 @@ jobs:
         if: env.FAIL_TASK == 'true'
         run: exit 1
 
+  check_program_compilation:
+    name: Checks if the library compiles correctly and binary files got created
+    runs-on: ubuntu-latest
+    needs: [check_repository_cleanliness]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Vulkan SDK
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libvulkan-dev
+
+      - uses: actions/cache@v3
+        name: Store XMake repositories in cache
+        with:
+          path: ~/.xmake
+          key: ${{ runner.os }}-xmake-entt-vulkan-glm
+          restore-keys: |
+            ${{ runner.os }}-xmake-entt-vulkan-glm
+
+      - name: Install xmake
+        run: |
+          curl -fsSL https://xmake.io/shget.text | bash
+
+      - name: (Try to) build the project
+        run: xmake build -y
+        timeout-minutes: 2
+
+      - name: Verifies that files are present and executable
+        run: |
+          echo "FAIL_TASK=false" >> "$GITHUB_ENV"
+          SEARCH_BINARIES="${{ env.EXECUTABLES }}"
+          IFS=$','
+          for BIN in $SEARCH_BINARIES; do
+            if [ ! -f "${BIN}" ]; then
+              echo "::error file=${BIN},title=Binary not found::${BIN}"
+              echo "FAIL_TASK=true" >> "$GITHUB_ENV"
+            fi
+            if [ ! -x "${BIN}" ]; then
+              echo "::error file=${BIN},title=Binary not executable::${BIN}"
+              echo "FAIL_TASK=true" >> "$GITHUB_ENV"
+            fi
+          done
+
+      - name: Fail if needed
+        if: env.FAIL_TASK == 'true'
+        run: exit 1
+
+  run_tests:
+    name: Run the program tests (using rule make tests_run)
```

Comment:
it's a copy of an old project I think because we don't use Makefile for the moment

Reply:
its only a typo